examples/04_pong/main.rs

```diff
@@ -335,6 +338,20 @@ impl State for Pong {
                 bounce_sfx,
                 score_sfx,
             });
+            // Sets the cursor hidden when the mouse is hovering the pong window.
+            if let Some(mut window_events) = engine
+                .world
+                .res
+                .try_fetch_mut::<EventHandler<WindowModifierEvent>>(0)
+            {
+                window_events.write_single(WindowModifierEvent {
+                    modify: Box::new(|win| {
+                        win.set_cursor_state(winit::CursorState::Hide)
+                            .ok()
```

Comment:
1) Why call `.ok` just to unwrap later?
2) Didn't we agree on not exposing `winit` stuff or is my memory wrong?
3) Those are 14 lines for something that basic, so shouldn't we change something here?
4) @Rhuagh You previously said that you're opposed to callbacks, but that's essentially what this PR is introducing, isn't it?
5) Should we eventually discuss a strategy for things like `Factory`, `Window`, ..?

Reply:
Yes, I'm opposed to callbacks, which this is not, this is essentially a Command. I do think we can do this in a number of different ways, in a more data-driven way, but I'm not sure if it's worth it.

Yes, we should discuss a strategy for framework local resources that the user needs to be able to control, but have no direct access to.

Reply:
I delete the whole thing, we keep it, or we modify it? :P